queries/models/dashboard_subsidio_sppo/CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Changelog - dashboard_subsidio_sppo
 
+## [8.0.5] - 2025-10-28
+
+### Corrigido
+
+- Corrigida a classificação de viagens acima do limite o modelo `viagens_remuneradas_v2` (https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/1005)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)